qiskit/pulse/builder.py

```diff
@@ -1530,11 +1519,23 @@ def play(
     append_instruction(instructions.Play(pulse, channel, name=name))
 
 
+_MetaDataType = TypedDict(
```

Comment:
Prefer the [class syntax](https://docs.python.org/3/library/typing.html#typing.TypedDict):
```python
class _MetaDataType(TypedDict):
    kernel: configuration.Kernel | None
    ...
```

Reply:
done!